osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/infra/jms/core/OsgpCoreResponseMessageProcessor.java

```diff
@@ -174,12 +177,11 @@ protected abstract void handleMessage(DeviceMessageMetadata deviceMessageMetadat
      * of defining the response at its own.
      *
      * @param e
-     *            the exception.
+     *         the exception.
      * @param deviceMessageMetadata
-     *            the device message metadata.
+     *         the device message metadata.
      * @param responseMessage
-     *            the response message.
-     * @throws FunctionalException
+     *         the response message.
      */
     protected void handleError(final Exception e, final DeviceMessageMetadata deviceMessageMetadata,
             final ResponseMessage responseMessage) throws FunctionalException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.FunctionalException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)